<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectUI</class>
 <widget class="QWidget" name="ProjectUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>title</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Contact Info:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="contact">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bmodifyContact">
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Link to project:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="url">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bmodifyUrl">
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QTextEdit" name="description">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>130</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bmodifyDescription">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Features:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextEdit" name="features"/>
     </item>
     <item>
      <widget class="QPushButton" name="bmodifyFeatures">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Instructor Comment:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="comment">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bmodifyComment">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="bdelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bmodifyContact</sender>
   <signal>clicked()</signal>
   <receiver>ProjectUI</receiver>
   <slot>modifyContact()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmodifyUrl</sender>
   <signal>clicked()</signal>
   <receiver>ProjectUI</receiver>
   <slot>modifyUrl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmodifyDescription</sender>
   <signal>clicked()</signal>
   <receiver>ProjectUI</receiver>
   <slot>modifyDescription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmodifyFeatures</sender>
   <signal>clicked()</signal>
   <receiver>ProjectUI</receiver>
   <slot>modifyFeatures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmodifyComment</sender>
   <signal>clicked()</signal>
   <receiver>ProjectUI</receiver>
   <slot>modifyComment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bdelete</sender>
   <signal>clicked()</signal>
   <receiver>ProjectUI</receiver>
   <slot>toConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifyContact()</slot>
  <slot>modifyUrl()</slot>
  <slot>modifyDescription()</slot>
  <slot>modifyFeatures()</slot>
  <slot>modifyComment()</slot>
  <slot>toConfirm()</slot>
 </slots>
</ui>
